Backend/Interview.Backend/RoomReviews/RoomReviewController.cs

```diff
@@ -57,15 +57,16 @@ public Task<IPagedList<RoomReviewPageDetail>> FindPage([FromQuery] RoomReviewPag
     }
 
     /// <summary>
-    /// Creating a review for a room.
+    /// Creating a review for a room [DEPRECATED]
```

Comment:
Use the “Obsolete” attribute with a comment, this will be correct, for the deprecated method.

Reply:
Ok.